### ISSUE TITLE
Labor Status Forms: prevent submitters from choosing certain WLS levels

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/lsf#11]
**[Original report](https://bitbucket.org/laborstudents/lsf-flask/issue/11) by Katelyn Adams (Bitbucket: [paesh1997](https://bitbucket.org/paesh1997), GitHub: [paesh1997](https://github.com/paesh1997)).**

----------------------------------------

If a student will be on WLS 5 or WLS 6, they should have at least a 15-hour contract \(10 and 12 are not allowed\)

If a student will hold a secondary position, they should be at most on WLS 4 \(WLS 5 and WLS 6 are not allowed\)

**Comment by bryantal** [comment]
**Original comment by Hila Manalai (Bitbucket: [Hila Manalai](https://bitbucket.org/Hila Manalai), ).**

----------------------------------------

@{5b1e7d606116e24d2bfc881d} Luis and I added a modal that checks for WLS, and hours when Add Student button is clicked. If WLS is greater than 5 and Hours are less than 15, then it shows a modal, and it doesn’t add the student to the table.

The modal language might need some modifications.

**Comment by bryantal** [comment]
**Original comment by Hila Manalai (Bitbucket: [Hila Manalai](https://bitbucket.org/Hila Manalai), ).**

----------------------------------------

Fixed issue 11 and 31

**Comment by bryantal** [comment]
**Original comment by Katelyn Adams (Bitbucket: [paesh1997](https://bitbucket.org/paesh1997), GitHub: [paesh1997](https://github.com/paesh1997)).**

----------------------------------------

@{5cfe56a6765c560c5d88ba1d} @{5cfe5696b287090c5cbedf55} Hows this one doing?

**Comment by bryantal** [comment]
**Original comment by Katelyn Adams (Bitbucket: [paesh1997](https://bitbucket.org/paesh1997), GitHub: [paesh1997](https://github.com/paesh1997)).**

----------------------------------------

alex & luis took LSF after Hila became a 0-5

**Comment by bryantal** [comment]
**Original comment by Luis Riera (Bitbucket: [Luis Riera](https://bitbucket.org/Luis Riera), ).**

----------------------------------------

This will likely be one of the last things we work on once the JS and controller are modified to account for the global Array of students. Thus far, I have not thought about it much.

**Comment by bryantal** [comment]
**Original comment by Luis Riera (Bitbucket: [Luis Riera](https://bitbucket.org/Luis Riera), ).**

----------------------------------------

Will be resolved once the refactoring of LSF’s JavaScript is merged into development \(issue #68\).

**Comment by bryantal** [comment]
**Original comment by Elaheh Jamali (Bitbucket: [elaJamali](https://bitbucket.org/elaJamali), GitHub: [Jamaliela](https://github.com/Jamaliela)).**

----------------------------------------

PR has been created and it is waiting to be reviewed.

**Comment by bryantal** [comment]
**Original comment by Elaheh Jamali (Bitbucket: [elaJamali](https://bitbucket.org/elaJamali), GitHub: [Jamaliela](https://github.com/Jamaliela)).**

----------------------------------------

The PR related to this issue has been successfully merged.
